client/components/site-verticals-suggestion-search/index.jsx

```diff
@@ -83,116 +154,74 @@ export class SiteVerticalsSuggestionSearch extends Component {
 				verticalSlug: value,
 			}
 		);
+	};
 
-	onSiteTopicChange = ( value, isNavigating ) => {
-		const hasValue = !! value;
-		const valueLength = value.length || 0;
-		const valueLengthShouldTriggerSearch = valueLength >= this.props.charsToTriggerSearch;
-		const result = this.searchForVerticalMatches( value );
-
-		// TODO:
-		// Where to put the railcar code will be reconsidered.
-		if (
-			hasValue &&
-			valueLengthShouldTriggerSearch &&
-			// Don't trigger a search if there's already an exact, non-user-defined match from the API
-			! result
-		) {
+	/**
+	 * Callback to be passed to consuming component when the search field is updated.
+	 *
+	 * @param {String} value The new search value
+	 * @param {Bool} isSuggestionSelected Whether the user has selected a suggestion
```

Comment:
Nit of the nits: I've found that vertically aligning the different fragments of the JSdoc makes it so much more readable:

```
	 * @param {String} value                The new search value
	 * @param {Bool}   isSuggestionSelected Whether the user has selected a suggestion
```

I'm fine either way tho - feel free to take this suggestion or throw it in the trash.